src/openrct2-ui/CMakeLists.txt

```diff
@@ -35,6 +35,11 @@ file(GLOB_RECURSE OPENRCT2_UI_HEADERS
 if (APPLE)
     file(GLOB_RECURSE OPENRCT2_UI_MM_SOURCES "${CMAKE_CURRENT_LIST_DIR}/*.mm")
     set_source_files_properties(${OPENRCT2_UI_MM_SOURCES} PROPERTIES COMPILE_FLAGS "-x objective-c++ -fmodules")
+    if ("${CMAKE_OSX_ARCHITECTURES}" MATCHES "arm64")
+        # cross-compilation workaround for SDL2
```

Comment:
Can you provide upstream ticket link here?

Reply:
Yes - let me update the comment. It's been resolved upstream, so we just need to wait for a new release, and then bump the vcpkg SDL2 version. We could add a vcpkg patch with the same change (until there is a new release), but this workaround is quite clean so I'm not sure it's worth creating a patch for it.